data/successors/successor 1 introduction.txt

```diff
@@ -169,7 +169,7 @@ mission "Successors: First Contact 1"
 				`	"What is House Sioeora?"`
 					goto house
 			label house
-			`	Sieasej translates your query to the military officer this time, who considers for a moment before saying something back and Sieasej speaks again. "Sioeora is the noble house whose estate thou sees around thee. There are many such within the broader realm; the stars this side'f the wormhole, but 'tis Sioeora who has taken patronage on thee, for the keeping of our history is the... core of its purpose. 'Twere thee apprehended by the arms of another - " Sieasej gestures towards an interceptor landed on another pad " - thou surely would have languished without recognition as a creature from so distant in our past."`
+			`	Sieasej translates your query to the military officer this time, who considers for a moment before saying something back and Sieasej speaks again. "Sioeora is the noble house whose estate thou sees around thee. There are many such within the broader realm; the stars this side'f the wormhole, but 'tis Sioeora who has taken patronage on thee, for the keeping of our history is the... core of its purpose. 'Twere thee apprehended by the arms of another - " Sieasej gestures towards an interceptor landed on another pad "- thou surely would have languished without recognition as a creature from so distant in our past."`
```

Comment:
So it looks like the checks fail if you have the space where I took it out. Otherwise, they work fine. So do you have thoughts on that?

Reply:
Hmm, I can see why this throws an error even though it probably shouldn't. Another reason I yearn for Unicode support. I think the best solution would be to just rephrase the sentence so fighting the checker isn't necessary, so I'll make a note to do that.